build.bash

```diff
@@ -100,7 +109,7 @@ check_command_availability_and_exit() {
 
 # mount rpi image using userspace tools, in docker use privileged mount via kpartx
 mount_image_file_boot() { # imagefile buildfolder
-  if ! running_in_docker && ! running_on_github && ! is_pi; then
+  if ! running_in_docker && ! running_on_github; then
```

Comment:
why removing the is_pi here? I think the guestmount package which is based on qemu/libvirt which is not available on the PI?

Reply:
the intention was to use "guestmount" on normal Linux PCs (in user mode), and for all others (including the RPI) the classic losetup/kpartx way (root required).... 
Do you actually build the images on a PI?

Reply:
yes of course

Reply:
I have the feeling that using classic ("root") way is better suited for building on the PI. Typical installations do not provide libguestfs-tools. For my production system, this would need ~600MB of extra packages to install.
But actually, I do not mind at all. Keep it or remove it, as you like....

Reply:
I wasn't aware of that so changed it back to build as root